convention-develocity-maven-extension/extension/src/main/java/com/myorg/ConventionDevelocityMavenExtensionListener.java

```diff
@@ -5,16 +5,10 @@
 import com.gradle.develocity.agent.maven.api.cache.BuildCacheApi;
 import com.gradle.develocity.agent.maven.api.scan.BuildScanApi;
 import org.apache.maven.execution.MavenSession;
-import org.codehaus.plexus.component.annotations.Component;
 
 /**
  * An example Maven extension for enabling and configuring Develocity features.
  */
-@Component(
-        role = DevelocityListener.class,
-        hint = "convention-develocity-maven-extension",
-        description = "Configures the Develocity Maven extension for com.myorg"
-)
 public final class ConventionDevelocityMavenExtensionListener implements DevelocityListener {
```

Comment:
It would be good to have a consistent naming pattern across all these listener implementations.

- I suggest renaming to simply `ConventionDevelocityListener`
- The CCUD listeners in the other PR LGTM

Reply:
Addressed in c0e0d84fe2508a1bafab1880d0f0fb9e9ada7c69